pkg/library/mappers/gateway.go

```diff
@@ -49,6 +49,7 @@ func (m *gatewayEventMapper) MapToPolicy(ctx context.Context, obj client.Object,
 		kuadrantgatewayapi.WithRoutes(utils.Map(routeList.Items, ptr.To[gatewayapiv1.HTTPRoute])),
 		kuadrantgatewayapi.WithPolicies(policies),
 		kuadrantgatewayapi.WithLogger(logger),
+		kuadrantgatewayapi.WithProgrammedGatewaysOnly(false),
```

Comment:
This should stop the filtering of gateway events based on programmed status for *all* policies right?

Reply:
Yes, correct